app/controllers/podcasts_controller.rb

```diff
@@ -1,6 +1,10 @@
 class PodcastsController < ApplicationController
   before_action :authenticate_user!
 
+  # skip Bullet on create as it currently triggers an error inside the rolify
+  # method "current_user.add_role()" we have no control of
+  around_action :skip_bullet, only: %i[create], if: -> { defined?(Bullet) }
```

Comment:
Thats annoying, OSS PR maybe??? 😉

Reply:
Unfortunately rolify seems to have been abandoned (which is kinda worrisome as we depend on it a lot, but it works...): 

- latest release is from December 2017: https://rubygems.org/gems/rolify
- also check https://github.com/RolifyCommunity/rolify/issues/523

I think it's more in a feature freeze, there are more recent commits since 2017 but they haven't been released

Reply:
Yikes. Definitely worth noting big time.